Furball.Vixie/Graphics/Backends/OpenGL41/QuadRendererGL41.cs

```diff
@@ -75,7 +75,7 @@ private struct InstanceData {
             this._boundTextures = new TextureGL41[this._backend.QueryMaxTextureUnits()];
 
             string vertSource = ResourceHelpers.GetStringResource("ShaderCode/OpenGL41/InstancedRenderer/VertexShader.glsl");
-            string fragSource = ResourceHelpers.GetStringResource("ShaderCode/OpenGL41/InstancedRenderer/FragmentShader.glsl");
+            string fragSource = ShadersGL41.GetFragment(backend);
```

Comment:
The entire ShadersGL*.GetFragment is a little bit too unclear, like what is this loading? what fragment shader is this loading, this needs to be way more specific than just GetFragment, i didnt know what it was doing till i looked it up either